### PR TITLE
Resolve cldrjs to 0.4.4 in bower.json

### PR DESCRIPTION
**Pull request by cpennington**
@mulby, @dsjen: Review?

### CODE REVIEW

**Review by dsjen — APPROVED** [review]
Looks great! Please wait until tests pass before merging. 👍

**Comment by mulby** [comment]
@thallada can you review this as well?

File: scripts/runTests.sh [hunk 1]
```diff
@@ -21,8 +21,8 @@ export ENABLE_OAUTH_TESTS=False
 export ENABLE_ERROR_PAGE_TESTS=False
 export ENABLE_LEARNER_ANALYTICS=True
 
-echo "Migrating Analytics Dashboard DB..."
-make migrate
+# echo "Migrating Analytics Dashboard DB..."
```
Comment (dsjen) [inline scripts/runTests.sh]: Did this cause a problem?

Comment (cpennington) [reply scripts/runTests.sh]: It was something that Tyler had fixed on master when dealing with the API checkout build, I think.

Comment (dsjen) [reply scripts/runTests.sh]: I think it would make sense to remove the commented out lines--matches more with what's on master.

**Review by dsjen — APPROVED** [review]
Recommendation to remove commented lines--not a blocker.  👍

**Comment by thallada** [comment]
Awesome :+1: thank you for hunting down the selenium error.

**Review by thallada — APPROVED** [review]

**Pull request merged**